pkg/api/README.md

```diff
@@ -4,10 +4,17 @@ Manager service is an API for core component.
 Suggestion service is an API for Suggestion services.
 EarlyStopping service is an API for EarlyStopping services.
 
-## Documentation
-Please refer to [api.md](./gen-doc/api.md).
+## v1alpha1 Documentation
```

Comment:
Probably, just name it `Documentation`? Since we have v1alpha1 and v1alpha2 links here.